.github/workflows/deploy.yml

```diff
@@ -0,0 +1,19 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+    - '*'
```

Comment:
@fortmarek should this be triggered every Friday at 10:00 and let Tapestry do the tagging?

Reply:
`tapestry github-release version` is the command that should be triggered continuously - but I'd first do one or two releases manually to validate that the process is working as intended and then we can take the automation one step further